### PR TITLE
feat: harness-pair-dev — reviewer-optional pair authoring

### PR DESCRIPTION
**Pull request by KingGyuSuh**
## Summary

Make `/harness-pair-dev` pair-by-default and reviewer-less opt-in, with the runtime, authoring rubric, and docs all aligned around that shape. Targets v0.1.5; `[Unreleased]` → `[0.1.5]` rename, badge bump, and `git tag` are deferred to a separate release-time commit after this PR merges.

### Breaking change

- `/harness-pair-dev --add` now takes `<purpose>` as a **positional** second argument. The `--purpose "<text>"` flag form is gone. Re-run callers as `/harness-pair-dev --add <pair-slug> "<purpose>" [--reviewer <slug>|none ...]`.

### New

- `--reviewer none` reviewer-less opt-in. Pair authoring stays the default; passing the literal `none` (and only that, never mixed with real reviewer slugs) produces a producer-only group with no reviewer agent file. `register-pair.ts` writes a third registration shape: `` - <pair>: producer `<p>` (no reviewer), skill `<s>` ``.

### Removed surface

- User-facing `--target` and `--provider` flags removed from `/harness-pair-dev`. Target is fixed to cwd; provider sync is delegated to `sync.ts` on-disk detection (claude always present; codex/gemini only when their derived trees already exist). To enable a new provider for the first time, call `/harness-sync --provider <list>` separately. The internal `register-pair.ts` script still accepts `--target` for integration tests.

### Runtime

- `harness-orchestrate` template recognizes the producer-only registration shape and dispatches reviewer-less producer turns cleanly. The missing `↔` arrow plus the literal `(no reviewer)` token are the load-bearing markers. Reviewer-less is framed as **"not subject to review"**, never as **"passed without review"**. A producer `## Structural Issue` block remains the only retreat trigger.

### Authoring rubric

- New `### 7. When reviewer-less is appropriate` section in `harness-pair-dev/SKILL.md`, plus matching Evaluation Criteria and Taboos. Scopes `--reviewer none` to deterministic / auxiliary work (sync, format, mirror, mechanical translation) and explicitly forbids it for creative / judgment / generative work (code authoring, doc writing, marketing copy, planning, schema design).

### Docs / translations

- `README.md` and four `docs/README.{es,ja,ko,zh-CN}.md` translations updated for the new positional `<purpose>`, `--reviewer none` example, and the removal of `--target` / `--provider`.

## Test plan

- [x] `node --test tests/*.test.mjs` — 34/34 pass locally (includes the new `tests/runtime-template-anchors.test.mjs` pinning the load-bearing runtime tokens).
- [ ] Manual smoke: `/harness-pair-dev --add harness-foo "spec foo edge cases"` produces a pair with reviewer.
- [ ] Manual smoke: `/harness-pair-dev --add harness-bar "mirror canonical assets" --reviewer none` produces a producer-only group and registers it with the `(no reviewer)` token.

### CODE REVIEW

File: plugins/harness-loom/skills/harness-init/references/runtime/harness-orchestrate/SKILL.template.md [hunk 1]
```diff
@@ -49,40 +62,43 @@ The execution order of one orchestrator response is fixed:
...
+1. **Rework** — aggregated verdict is FAIL (or `any FAIL` in 1:M; or producer self-`Status: FAIL` in reviewer-less):
    - `Next.To` = the same producer
    - `Next.EPIC` = the same EPIC
-   - `Next.Intent` = `Rework (reviewer FAIL): <summary of FAIL reasons>`; in 1:M merge reviewer reasons with axis tags
+   - `Next.Intent` = `Rework (reviewer FAIL): <summary of FAIL reasons>` for paired turns; `Rework (producer FAIL): <summary of producer-stated FAIL reasons>` for reviewer-less turns. In 1:M merge reviewer reasons with axis tags.
    - `Next.Prior tasks` = `[the task file just written]`
-   - `Next.Prior reviews` = `[all review files that failed]`, plus PASS review files if needed so already-passing axes are preserved
-2. **Retreat (structural)** — producer or reviewer reports a structural issue:
+   - `Next.Prior reviews` = `[all review files that failed]` for paired turns, plus PASS review files if needed so already-passing axes are preserved; `[]` (empty) for reviewer-less turns since no review file was written
+2. **Retreat (structural)** — producer or reviewer reports a structural issue (for reviewer-less turns this can only come from the producer's own artifact):
```
[the remark this answers is not in the document]
Comment (KingGyuSuh) [reply plugins/harness-loom/skills/harness-init/references/runtime/harness-orchestrate/SKILL.template.md:94]: Addressed in c35ec3e. Phase advance Retreat rule (line 99 — the `Prior reviews` field, not the rule's intro at line 94) now mirrors the Rework rule's pattern: `[the structural issue review]` for paired turns, `[]` (empty) for reviewer-less turns. The structural issue lives in the producer's task file, which is already in `Next.Prior tasks`, so no review-file artifact needs to be synthesized for the retreat envelope. node:test 34/34 PASS.
